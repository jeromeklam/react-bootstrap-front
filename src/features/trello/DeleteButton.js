import React from 'react'

const DeleteButton = props => {
  return (
    <div className="trello-delete-button" {...props}>
      <button className="btn btn-xs text-danger">&#10006;</button>
    </div>
  )
}

export default DeleteButton
