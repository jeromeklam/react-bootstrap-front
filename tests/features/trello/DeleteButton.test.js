import React from 'react';
import renderer from 'react-test-renderer';
import DeleteButton from '../../../src/features/trello/DeleteButton';

const callback = jest.fn();
jest.useFakeTimers();

it('render with className and name', () => {
  // First render
  const component = renderer.create(
    <DeleteButton className="btn bg-secondary" onClick={callback}>
      My Button
    </DeleteButton>
  );
  let test = component.toJSON();
  expect(test).toMatchSnapshot();
});
