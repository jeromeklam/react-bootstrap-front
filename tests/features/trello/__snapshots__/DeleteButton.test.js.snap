// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render with className and name 1`] = `
<div
  className="btn bg-secondary"
  onClick={[MockFunction]}
>
  <button
    className="btn btn-xs text-danger"
  >
    âœ–
  </button>
</div>
`;
